jdk/src/share/classes/java/lang/Shutdown.java

```diff
@@ -121,13 +128,20 @@ private static void runHooks() {
                     hook = hooks[i];
                 }
                 if (hook != null) hook.run();
-            } catch(Throwable t) {
+            } catch (Throwable t) {
                 if (t instanceof ThreadDeath) {
                     ThreadDeath td = (ThreadDeath)t;
                     throw td;
                 }
             }
         }
+
+        // set shutdown state
+        // Synchronization is for visibility; only one thread
+        // can ever get here.
+        synchronized(lock) {
```

Comment:
```suggestion
        synchronized (lock) {
```

Nit: a space after `synchronized`, consistent with the style in this local file.

Reply:
Fixed.